.changeset/great-sloths-rule.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Fixes a bug where Server Island wouldn't work when rendered inside a Fragment component.
```

Comment:
```suggestion
Fixes a bug where Server Islands wouldn't work when rendered inside a Fragment component.
```

Reply:
I changed the wording. Please review again @sarah11918  and @ascorbic